content/hi/docs/setup/best-practices/multiple-zones.md

```diff
@@ -0,0 +1,124 @@
+{{< /note >}}
+
+## नोड व्यवहार
+
+कुबेरनेट्स स्वचालित रूप से पॉड्स को फैलाता है
+कार्यभार संसाधन (जैसे {{< glossary_tooltip text="डिप्लॉयमेंट(Deployment)" term_id="deployment" >}} or {{< glossary_tooltip text="स्टेटफुलसेट(StatefulSet)" term_id="statefulset" >}})
+एक क्लस्टर में विभिन्न नोड्स में। यह प्रसार मदद करता है
+विफलताओं के प्रभाव को कम करना।
+
+जब नोड शुरू होता है, तो प्रत्येक नोड पर क्यूबलेट स्वचालित रूप से नोड ऑब्जेक्ट के लिए {{< glossary_tooltip text="labels" term_id="label" >}} जोर देता है जो कुबेरनेट्स एपीआई में उस विशिष्ट क्यूबलेट का प्रतिनिधित्व करता है।
+इन लेबल में शामिल हो सकते हैं
+[क्षेत्र की जानकारी](/docs/reference/labels-annotations-taints/#topologykubernetesiozone).
+
+यदि आपका क्लस्टर कई प्रदेश या क्षेत्रों में फैला है, तो आप नोड लेबल का उपयोग 
+[पॉड टोपोलॉजी फैलाव में प्रतिबंध](/docs/concepts/workloads/pods/pod-topology-spread-constraints/) के साथ संयोजन के रूप में कर सकते हैं।
+यह नियंत्रित करने के लिए कि आपके क्लस्टर में फॉल्ट डोमेन के बीच पॉड कैसे फैले हैं:
+प्रदेश, क्षेत्रों, और यहां तक ​​कि विशिष्ट नोड्स।
+
+ये संकेत {{< glossary_tooltip text="scheduler" term_id="kube-scheduler" >}} 
+बेहतर अपेक्षित उपलब्धता के लिए पॉड्स को जगह देने में सक्षम बनाते हैं, सहसंबद्ध जोखिम को कम करते हैं क्योंकि
+विफलता आपके पूरे कार्यभार को प्रभावित करती है।
+
+उदाहरण के लिए, आप यह सुनिश्चित करने के लिए एक बाधा सेट कर सकते हैं कि
+जब भी संभव हो, स्टेटफुलसेट की 3 प्रतिकृतियां अलग-अलग क्षेत्रों में चल रही हैं। आप इसे घोषणात्मक रूप से परिभाषित कर सकते हैं
+स्पष्ट रूप से परिभाषित किए बिना कि प्रत्येक वर्कलोड कौन से उपलब्धता क्षेत्र के उपयोग में हैं।
+
+### क्षेत्रों में नोड्स वितरित करना
+
+कुबेरनेट्स का कोर आपके लिए नोड नहीं बनाता है, आपको इसे स्वयं करने की आवश्यकता है,
+या अपनी ओर से नोड्स को प्रबंधित करने के लिए [क्लस्टर एपीआई](https://cluster-api.sigs.k8s.io/) जैसे टूल का उपयोग करें।
+
+क्लस्टर एपीआई जैसे उपकरणों का उपयोग करके आप कई विफलता डोमेन में अपने क्लस्टर के लिए कार्यकर्ता नोड्स के रूप में चलाने के लिए मशीनों के सेट को परिभाषित कर सकते हैं यह नियम पूरे क्षेत्र में सेवा बाधित होने की स्थिति में क्लस्टर को स्वचालित रूप से ठीक कर देता है।
+
+## पॉड्स के लिए मैनुअल ज़ोन असाइनमेंट
+
+आप [नोड चयनकर्ता बाधाओं] को 
+आपके द्वारा बनाए गए पॉड्स में लागू कर सकते हैं के साथ-साथ वर्कलोड संसाधनों में पॉड टेम्प्लेट के लिए जैसे डिप्लॉयमेंट, स्टेटफुलसेट, या नौकरी में लागू कर सकते हैं।
+
+## क्षेत्रों के लिए संग्रहण पहुंच
+
+जब लगातार वॉल्यूम बनाए जाते हैं, तो `PersistentVolumeLabel`
+[प्रवेश नियंत्रक](/docs/reference/access-authn-authz/admission-controllers/)
+स्वचालित रूप से किसी भी PersistentVolumes को ज़ोन लेबल में जोड़ता है जो किसी विशिष्ट क्षेत्र से जुड़े होते हैं। तब {{< glossary_tooltip text="scheduler" term_id="kube-scheduler" >}} सुनिश्चित करता है कि अपने `NoVolumeZoneConflict` विधेय के माध्यम से, वह पॉड्स जो किसी दिए गए PersistentVolume का दावा करते हैं
+केवल उस वॉल्यूम के समान क्षेत्र में रखे जाते हैं।
+
+आप एक {{< glossary_tooltip text="StorageClass" term_id="storage-class" >}} निर्दिष्ट कर सकते हैं
```

Comment:
If localizing Deployment to डिप्लॉयमेंट (see earlier) then I think we should be consistent and also transcribe StorageClass; these are both API kinds. Maybe add an explanation as well: something like (भंडारण का वर्ग).